doc/gov/chapters/chapter09.md

```diff
@@ -1,135 +1,157 @@
 
+The transition plan described here informs application designers on how the Reference Compliance, and ultimately the industry certification programs will manage and document exceptions encountered during the qualification process.  The actions taken might include flagging warnings and potential errors caused by the variance from the CNTT conformance levels, which could prevent issuance of a certification badge.
 
-<p align="center"><img src="../figures/vnf_cnf_transition.png" alt="Transition" title="VNF/CNF Transition Plan" width="70%"/></p>
-<p align="center"><b>Figure 1:</b> Transition Plan for VNFs/CNFs within CNTT</p>
+<a name="9.4.1"></a>
+### 9.4.1 Conformance Levels
+
+- **Fully Conformant**: VNFs/CNFs or NFVI written and designed to be fully conformant to CNTT specifications with no use of any of the allowed Exceptions.
+- **Conformant with Exceptions**: VNFs/CNFs or NFVI written and designed to be conformant to CNTT specifications with one or more of the allowed Exceptions used.
+
+<a name="9.4.2"></a>
+### 9.4.2 Exception Types
+
+- **Technology Exceptions** : The use of specific technologies that are considered non conformant to CNTT principles (such as PCIe Direct Assignment, exposure of hardware features to VNFs/CNFs).
+- **Version Exceptions**: Using versions of  Software components, , APIs, or Hardware that are different from the ones specified in the specification.
+
+<a name="9.4.3"></a>
+### 9.4.3 Transition Framework
 
 #### NFVI Transition Plan Framework 
 
-Exceptions will be clearly recorded in Reference Architectures' Appendices which will act as a guidance to NFVI vendors of what Exceptions will be allowed in each CNTT release. **Figure 2** below demonstrate the concept.
+Exceptions will be clearly recorded in a given Reference Architecture's Appendix.  That document provides guidance to NFVI vendors of what Exceptions will be allowed in each CNTT release. **Figure 2** below demonstrates the concept.
 
-- As technology matures, fewer and fewer Exceptions will be allowed in CNTT releases.
-- For each CNTT Release, VNF/CNF can be either:
-  - **Fully Conformant**: Support the Target Reference Architecture without any exceptions. There should be a technology choice in RA to support RM Exceptions (However, none of the Exceptions allowed in RA has been used).
+- It is expected that over time, as technology matures, there will be a decreasing numbers of Exceptions allowed in CNTT releases.
+- For each CNTT Release, the NFVI can be either:
+  - **Fully Conformant**: Supports the Target Reference Architecture without any exceptions. There should be a technology choice in RA to support RM Exceptions (However, none of the Exceptions allowed in RA has been used).
```

Comment:
Conformance was defined in 9.4.1 -- these definitions of Fully Conformant and Conformant with Exceptions are duplicate and should be deleted.